blob/blob.go

```diff
@@ -25,9 +25,12 @@ const ProtoIndexWrapperTypeID = "INDX"
 // MaxShareVersion is the maximum value a share version can be. See: [shares.MaxShareVersion].
 const MaxShareVersion = 127
 
+// SignerSize is the size of the signer in bytes.
+const SignerSize = 20
+
 // Blob (stands for binary large object) is a core type that represents data
 // to be submitted to the Celestia network alongside an accompanying namespace
-// and optional signer (for proving the signer of the blob)
+// and optional signer (for proving the author of the blob)
```

Comment:
[nit] I think this was accidental b/c we renamed it to signer in a different PR

```suggestion
// and optional signer (for proving the signer of the blob)
```